From 749d10d4c61a84d8ba506f178daededac8062c3f Mon Sep 17 00:00:00 2001
From: Lei Maohui <leimaohui@fujitsu.com>
Date: Wed, 16 Aug 2023 02:43:08 +0000
Subject: [PATCH] Fix build error as following:

| configure: error: in '/home/aarch64-test/tmp/work/aarch64-ubinux-linux/resource-agents/4.5.0-r0/build':
| configure: error: C preprocessor "aarch64-ubinux-linux-gcc -E --sysroot=/home/aarch64-test/tmp/work/aarch64-ubinux-linux/resource-agents/4.5.0-r0/recipe-sysroot   -fstack-protector-strong  -O2 -D_FORTIFY_SOURCE=2 -Wformat -Wformat-security -Werror=format-security" fails sanity check

Signed-off-by: Lei Maohui <leimaohui@fujitsu.com>
---
 configure.ac | 1 -
 1 file changed, 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index d682ad780..84444a525 100644
--- a/configure.ac
+++ b/configure.ac
@@ -883,7 +883,6 @@ else
 		-Wno-strict-aliasing
 		-Wpointer-arith 
 		-Wstrict-prototypes
-		-Wunsigned-char
 		-Wwrite-strings"
 
 # Additional warnings it might be nice to enable one day
-- 
2.34.1

